src/main/java/io/jenkins/plugins/audit/filter/RequestContextFilter.java

```diff
@@ -0,0 +1,35 @@
+package io.jenkins.plugins.audit.filter;
+
+import hudson.Extension;
+import hudson.init.Initializer;
+import hudson.init.Terminator;
+import hudson.util.PluginServletFilter;
+import jenkins.model.Jenkins;
+import org.apache.logging.log4j.ThreadContext;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import java.io.IOException;
+
+@Extension
```

Comment:
I don't think you need this annotation; that could be causing problems.

Reply:
The issue was related to my way of using the ThreadContext by mistake.